index.js

```diff
@@ -2,17 +2,92 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+/****notes****/
+// use defer in <script> tag in index.html. 
+//reads top to bottom, and index.js is above/before our <body><h1> header. 
+//defer will make it so it is read last.
+//dont need defer in data.js, since it does not rely on any html
+
+
 
 /***** Deliverable 2 *****/
 
+// const header = document.querySelector("#header")
+header.style.color = '#cbf21b'
+
+/****notes****/
+//use console.dir(header) to locate header in console and see the css for the header element
+//scroll down and see >style
+//there will be a color: "" in style. 
+//nested:
+//<h1> (header)
+//  <style>
+//    <color:>
+//once we know the nesting, we can then write the code in order of nesting
+// header.style.color = ' 'rgba(0,0,0,0)', '#hex', or 'written color' '
+
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+
+//document.querySelectorAll('.traveler')     ---what i had before review. 
```

Comment:
This will give us a nodeList containing one element (the div with the class of traveler). So you could grab the div from the nodeList using bracket notation (i.e. `const travelerDiv = document.querySelectorAll('.traveler')[0]`) and then search for the relevant elements from there. This isn't a bad start! However, your updated code is even better 💯